include/zephyr/net/http/hpack.h

```diff
@@ -94,6 +94,12 @@ enum http_hpack_static_key {
 /* TODO Kconfig */
 #define HTTP2_HEADER_FIELD_MAX_LEN 256
 
+#if defined(CONFIG_HTTP_SERVER)
+#define HTTP_SERVER_HUFFMAN_DECODE_BUFFER_SIZE CONFIG_HTTP_SERVER_HUFFMAN_DECODE_BUFFER_SIZE
+#else
+#define HTTP_SERVER_HUFFMAN_DECODE_BUFFER_SIZE 0
```

Comment:
Do we allow zero-length arrays in Zephyr? Would be better to avoid it I think?
https://github.com/zephyrproject-rtos/zephyr/issues/63845

Reply:
This is only used to compile test the struct where the array is used.